src/facade/reply_builder.cc

```diff
@@ -448,6 +451,22 @@ void RedisReplyBuilder::SendLabeledScoredArray(std::string_view arr_label, Score
   }
 }
 
+template <typename I> void RedisReplyBuilder::SendLongArr(absl::Span<const I> longs) {
+  static_assert(std::is_integral_v<I>, "Must use integral type");
+  ReplyScope scope(this);
+  StartArray(longs.size());
+  for (auto v : longs) {
+    if constexpr (std::is_unsigned_v<I>)
+      DCHECK_LE(uint64_t(v), uint64_t(std::numeric_limits<long>::max()));
```

Comment:
what do you check here? and in any case, pls use `assert` in header files we do not import logging there.